src/tooltip/Tooltip.tsx

```diff
@@ -27,33 +27,47 @@ export type TooltipProps = TooltipInitialState & {
    *
    * @default true
    */
-  arrow?: boolean;
+  hasArrow?: boolean;
```

Comment:
We are mostly not using `has`, `is` prefixes for our boolean props... so maybe `arrow` or `showArrow` would be appropriate name..

Reply:
Yay, showArrow would be fine.